cmd/ops/program.go

```diff
@@ -0,0 +1,32 @@
+package ops
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	//Global Flags
+	outputFormat string
+
+	//Binary program
+	Program = &cobra.Command{
+		Use:   "gql-lint",
+		Short: "gql-lint is a tool to lint GraphQL queries and mutations",
+	}
+)
+
+func init() {
+	Program.CompletionOptions.DisableDefaultCmd = true
+	Program.PersistentFlags().StringVar(&outputFormat, "output", "stdout", "Output format. Choose between json and stdout. Defaults is stdout.")
+
+	// TODO: This is required because the test suite doesn't finish the program and flags are not reset. Find a better way to do this.
```

Comment:
This is here just because the test don't end the program and to have global flags they need to be at the root level which doesn't get clean up. One option will be to make this flag command specific but so far, we are supporting both stdout and json output for all outputs that are not errors